imports/plugins/included/search-mongo/server/methods/searchcollections.js

```diff
@@ -134,76 +136,84 @@ export function ensureProductSearchIndex() {
   rawProductSearchCollection.createIndex(indexObject, weightObject, getSearchLanguage());
 }
 
-
-export function buildOrderSearch(cb) {
-  check(cb, Match.Optional(Function));
-  const shopId = Reaction.getShopId();
-  const orders = Orders.find({}).fetch();
-  for (const order of orders) {
-    const user = Meteor.users.findOne(order.userId);
-    const userEmails = [];
-    if (user) {
-      for (const email of user.emails) {
-        userEmails.push(email.address);
-      }
-    }
-    const orderSearch = {
-      _id: order._id,
-      shopId: shopId,
-      shippingName: order.shipping[0].address.fullName,
-      billingName: order.billing[0].address.fullName,
-      userEmails: userEmails
-    };
-    OrderSearch.insert(orderSearch);
-  }
-  const rawOrderSearchCollection = OrderSearch.rawCollection();
-  rawOrderSearchCollection.dropIndexes("*");
-  rawOrderSearchCollection.createIndex({shopId: 1, shippingName: 1, billingName: 1, userEmails: 1});
-  if (cb) {
-    cb();
-  }
-}
-
-export function buildOrderSearchRecord(orderId, cb) {
+export function buildOrderSearchRecord(orderId) {
   const order = Orders.findOne(orderId);
-  const shopId = Reaction.getShopId();
   const user = Meteor.users.findOne(order.userId);
   const userEmails = [];
   if (user) {
     for (const email of user.emails) {
       userEmails.push(email.address);
     }
   }
-  const orderSearch = {
-    _id: order._id,
-    shopId: shopId,
-    shippingName: order.shipping[0].address.fullName,
-    billingName: order.billing[0].address.fullName,
-    userEmails: userEmails
+  const orderSearch = {};
+  for (const field of requiredFields.orders) {
+    if (transformations.orders[field]) {
+      orderSearch[field] = transformations.orders[field](order[field]);
+    } else {
+      orderSearch[field] = order[field];
+    }
+  }
+  orderSearch.billingName = order.billing[0].address.fullName;
+  orderSearch.billingPhone = _.replace(order.billing[0].address.phone, /\D/g, "");
```

Comment:
He is using the results from the OrderSearch collection where I am stripping them out to make them more searchable.